feature-libs/tracking/personalization/root/config/personalization-config.ts

```diff
@@ -8,7 +8,7 @@ import { Config } from '@spartacus/core';
 export abstract class PersonalizationConfig {
   personalization: {
     enabled: boolean;
```

Comment:
I know we don't need to make this optional, but just asking to make sure

Previously 
line 10: enabled was optional
line 16: slotPosition was optional
line 17: componentId: was optional

Pretty sure the boolean one doesn't need to be optional

Reply:
`httpHeaderName` should not be `optional`. I fixed it in another ticket. If `context` is configured, then it must have `slotPostion` and `componentId`. So, `context` is optional, but `slotPostion` and `componentId` are not optional.